README.md

```diff
@@ -42,30 +42,45 @@ subcommand.
 
 Top-level help:
 
+<!-- [[[cog
+from scripts.run_command import run
+run("pypistats --help")
+]]] -->
+
 ```console
 $ pypistats --help
-usage: pypistats [-h] [-V] {recent,overall,python_major,python_minor,system} ...
+usage: pypistats [-h] [-V]
+                 {recent,overall,python_major,python_minor,system} ...
 
 positional arguments:
   {recent,overall,python_major,python_minor,system}
 
-optional arguments:
+options:
   -h, --help            show this help message and exit
   -V, --version         show program's version number and exit
 ```
 
+<!-- [[[end]]] -->
+
 Help for a subcommand:
 
+<!-- [[[cog
+from scripts.run_command import run
```

Comment:
Hi, 

just a little detail. From the official cog docs:

> If there are multiple generators in the same file, they are executed with the same globals dictionary, so it is as if they were all one Python module.

https://nedbatchelder.com/code/cog/index.html#h_installation

That means, there's no need to import `run` on each block. You can just use it directly, à la

`<!-- [[[cog run("pypistats recent --help")]]] -->`

Reply:
I just learned that on this podcast interview with Ned about Cog!

https://www.pythonpodcast.com/cog-python-text-generation-episode-347/

I'll update this, thanks for the ping!